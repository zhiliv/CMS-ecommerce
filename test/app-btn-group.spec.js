import { shallowMount } from '@vue/test-utils'
import H from '@/components/app/btn-group/btn-group.vue'

describe('app-btn-group.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(H, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent',
        size: 'vertical',
        role: 'group'
      },
      slots:{
        default: 'Содержимое'
      }
    })
  })
  it('Содержимое html для app-btn-group ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
