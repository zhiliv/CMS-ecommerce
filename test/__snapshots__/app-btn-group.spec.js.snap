// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-btn-group.js Содержимое html для app-btn-group  1`] = `
<div
  class="btn-group btn-group-vertical simple simple-xl"
  id="indent"
  role="group"
  style="background: white;"
>
  Содержимое
</div>
`;
